README.md

```diff
@@ -16,7 +16,7 @@
 Install conda (through [anaconda](https://docs.anaconda.com/anaconda/install/) or [miniconda](https://docs.conda.io/en/latest/miniconda.html)), then run:
 
 ```sh
-conda env create -f conda.yaml
+conda env update -f conda.yaml --prune
```

Comment:
doesn't update only work once env is created?

Reply:
Didn't try that, will update this to have two commands if needed (or find a create_or_update...)

Reply:
I checked and update works even when the env doesn't exist yet.